src/systems/a320_systems/src/lib.rs

```diff
@@ -162,6 +162,7 @@ impl Aircraft for A320 {
             &self.lgciu2,
             &self.emergency_electrical_overhead,
             &self.electrical,
+            &self.electrical,
```

Comment:
**thought**: Consider if combining these two as one argument `&(impl EmergencyElectricalState + EmergencyGeneratorInterface)` is better. I'm not sure. Pick whatever feels nice. Passing the same thing twice feels a bit odd, even though it probably is fine 😀.